store.go

```diff
@@ -2473,6 +2478,32 @@ func (s *store) Version() ([][2]string, error) {
 	return [][2]string{}, nil
 }
 
+func (s *store) MountImage(id string, mountOptions []string) (string, error) {
+	// Currently only ReadOnly option is set
+	// and value passed to mountOptions is ignored.
+	mountOptions = []string{"ro"}
```

Comment:
This should be appended on, we still want to pass in the mount options.  We should support NODEV, NOEXEC, NOSUID ...